google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -489,13 +490,17 @@ def delete force: nil
         #
         # @!group Table
         #
-        def create_table table_id, name: nil, description: nil
+        def create_table table_id, name: nil, description: nil, encryption_configuration: nil
           ensure_service!
+          unless encryption_configuration.nil?
+            encryption_configuration = encryption_configuration.to_gapi
+          end
           new_tb = Google::Apis::BigqueryV2::Table.new(
             table_reference: Google::Apis::BigqueryV2::TableReference.new(
               project_id: project_id, dataset_id: dataset_id,
               table_id: table_id
-            )
+            ),
+            encryption_configuration: encryption_configuration
```

Comment:
I would add this to the `Table::Updater` object instead of the `Google::Apis::BigqueryV2::Table` object. That way any validation logic present in `Table#encryption_configuration=` would also be run here.

Reply:
Oops, never mind. I was looking at an older commit and this looks to have been removed completely.

Reply:
Sorry, I think you ended up on an old commit. Now that I am using the Table Updater, I didn't need to make any changes to `create_table`.